tests/ui-fulldeps/stable-mir/check_transform.rs

```diff
@@ -137,9 +137,9 @@ fn generate_input(path: &str) -> std::io::Result<()> {
     write!(
         file,
         r#"
-        #![feature(panic_internals)]
```

Comment:
Was it really necessary to make this test use extremely unstable *internal* features? I hope we can avoid that, it just makes unnecessary work for others when those internal details get changed.

Reply:
Cc @celinval

Reply:
Fair.. sorry